examples/gridViewExample/src/main/java/io/realm/examples/realmgridview/City.java

```diff
@@ -19,7 +19,8 @@
 import io.realm.RealmObject;
 
 public class City extends RealmObject {
-
+    // Field names should not be obfuscated.
```

Comment:
Perhaps rephrase to `If you are using GSON, field names should not be obfuscated. Add either the proguard rule in proguard-rules.pro or the @SerializedName annotation`